cmd/erda-server/bootstrap.yaml

```diff
@@ -254,7 +254,7 @@ openapi-v1-routes:
           - msp-notify-list
           - msp-notify-detail
 openapi-dynamic-routes:
-  _enable: ${OPENAPI_DEV_MODE:false}
+  _enable: ${OPENAPI_DYNAMIC_ROUTES:false}
```

Comment:
maybe you should set this value at dice.yml

Reply:
done